front/src/components/profiles/Profiles.vue

```diff
@@ -102,9 +114,12 @@ export default {
       }
     },
     editProfile(profile) {
-      if (this.editing) {
+      if (this.editing && profile.pseudo != "Invité") {
         this.choosedProfile = profile;
         this.editingChoosedProfile = true;
+        this.inviteProfil = false;
+      } else {
+        this.inviteProfil = true;
```

Comment:
S'assurer que le inviteProfil est mis à jour uniquement lorsque this.editing est à true